### PR TITLE
MDN Feature Pages for SVGAnimatedAngle: animVal and baseVal 

### PR DESCRIPTION
**Pull request by yashrajbharti**
### Description

This PR adds the MDN feature pages for the SVGAnimatedAngle interface as part of a recent project to document missing Widely available pages for interface features. 

### Motivation

I believe that my support in documenting the SVGAnimatedAngle interface can significantly help developers by providing essential resources.

### Additional details

For more context, see the discussion on missing documentation tracked at https://openwebdocs.github.io/web-docs-backlog/baseline/. Additionally, related projects for adding widely available web features can be found at https://github.com/openwebdocs/project/issues/214. 

### Related issues and pull requests

Relates to [#214](https://github.com/openwebdocs/project/issues/214)

### CODE REVIEW

File: files/en-us/web/api/svganimatedangle/baseval/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const marker = document.createElementNS("http://www.w3.org/2000/svg", "marker");
+svg.appendChild(marker);
+document.body.appendChild(svg);
+
+// Set the orient attribute with an angle
+marker.setAttribute("orient", "90");
+const baseAngle = marker.orientAngle.baseVal;
+
+console.log(baseAngle.value); // Outputs 90
+console.log(baseAngle.unitType); // Outputs the unit type (e.g., SVG_ANGLETYPE_DEG)
```
Comment (estelle) [inline files/en-us/web/api/svganimatedangle/baseval/index.md:36]:
```suggestion
const baseAngle = marker.orientAngle.baseVal; // an SVGAngle object

console.log(baseAngle.value); // Output: 90
console.log(baseAngle.unitType); // Output: 1 (constant for SVG_ANGLETYPE_UNSPECIFIED)
```

File: files/en-us/web/api/svganimatedangle/index.md [hunk 2]
```diff
@@ -60,14 +60,14 @@ The `SVGAnimatedAngle` interface is used for attributes of basic type [\<angle>]
...
-      <td><code>baseVal</code></td>
+      <td><code>{{domxref("SVGAnimatedAngle.baseVal")}}</code></td>
       <td>{{ domxref("SVGAngle") }}</td>
       <td>
         The base value of the given attribute before applying any animations.
       </td>
     </tr>
     <tr>
-      <td><code>animVal</code></td>
+      <td><code>{{domxref("SVGAnimatedAngle.baseVal")}}</code></td>
```
Comment (estelle) [inline files/en-us/web/api/svganimatedangle/index.md:70]:
```suggestion
      <td><code>{{domxref("SVGAnimatedAngle.animVal")}}</code></td>
```

File: files/en-us/web/api/svganimatedangle/baseval/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/api/svganimatedangle/baseval/index.md:36]: same feedback for animVal

**Review by estelle — COMMENTED** [review]
Thank you. It looks good.

I gave some feedback that's relevant to this PR and also some for all API pages.

**Comment by yashrajbharti** [comment]
Thanks for all the wonderful suggestions! I have started fixing these in other pages and learning on the way.

File: files/en-us/web/api/svganimatedangle/animval/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- {{domxref("SVGAnimatedAngle.baseVal")}}
+- {{domxref("SVGAngle")}}
```
Comment (estelle) [inline files/en-us/web/api/svganimatedangle/animval/index.md:48]:
```suggestion
- {{domxref("SVGAngle")}} and [angle `value` constants](/en-US/docs/Web/API/SVGAngle#constants)
```

PR https://github.com/mdn/content/pull/37114 adds the constant number values.

File: files/en-us/web/api/svganimatedangle/baseval/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- {{domxref("SVGAnimatedAngle.animVal")}}
+- {{domxref("SVGAngle")}}
```
Comment (estelle) [inline files/en-us/web/api/svganimatedangle/baseval/index.md:46]:
```suggestion
- {{domxref("SVGAngle")}} and [angle `value` constants](/en-US/docs/Web/API/SVGAngle#constants)
```

**Review by estelle — COMMENTED** [review]
minor tweaks and good to go.

**Review by estelle — APPROVED** [review]
thanks!

**Pull request merged**